source/components/Bubble.js

```diff
@@ -110,34 +124,46 @@ class Bubble extends Component {
     }
   }
 
-  _getFirstScrollableParent = node => {
-    if (node == null) return null;
-    if (node === this.rootElement || node.scrollHeight <= node.clientHeight) {
-      return this._getFirstScrollableParent(node.parentNode);
-    } else {
-      return node;
+  _getFirstScrollableParent = (element: ElementRef<*>) => {
+    if (element == null) return null;
+    const { rootElement } = this;
+    const node = {}.hasOwnProperty.call(element, 'current') ? element.current : element;
+
+    if (rootElement) {
+      if (node === rootElement.current || node.scrollHeight <= node.clientHeight) {
+        return this._getFirstScrollableParent(node.parentElement);
+      }
     }
+
+    return node;
   };
 
   _updatePosition = () => {
     const { isOpeningUpward } = this.props;
-    const rootNode = this.rootElement;
-    const parentNode = rootNode.parentNode;
-    const parentNodeParams = parentNode.getBoundingClientRect();
-
-    let positionY;
-    if (isOpeningUpward) {
-      positionY = window.innerHeight - parentNodeParams.top + 20;
-    } else {
-      positionY = parentNodeParams.bottom + 20;
+    const { rootElement } = this;
+
+    if (rootElement) {
```

Comment:
Instead of wrapping the whole function in this conditional it would be nicer to just do that check in the beginning and throw an error if it does not exist (which is not expected here anyway!)

Reply:
Makes sense! Fixed it.